server/apiserver/apiserver.go

```diff
@@ -83,13 +82,13 @@ func NewApiServer(servers map[int]*common.ServerConfig, nodeId int) (*ApiServer,
 func (api *ApiServer) ListenAndServe() {
 	pool := NewPool()
 	go pool.Start(api.Mux)
-	http.HandleFunc("/get_image", api.HTTPGetImage)
-	http.HandleFunc("/json/image", api.HTTPGetImageJson)
+	// http.HandleFunc("/get_image", api.HTTPGetImage)
```

Comment:
Let's not just comment this out.  I'd prefer we keep some HTTP APIs for testing (specifically the JSON endpoint), but if you really want to get rid of them then remove the lines rather than commenting out.